src/components/Checkbox/__tests__/Checkbox.spec.js

```diff
@@ -0,0 +1,65 @@
+import { describe, expect, it } from 'vitest';
+import { mount } from '@vue/test-utils';
+
+import Checkbox from '../Checkbox.vue';
+
+const createWrapper = (props) => {
+  return mount(Checkbox, {
+    props,
+  });
+};
+
+describe('Checkbox', () => {
+  it('should render right label', () => {
+    const wrapper = createWrapper({ textRight: 'Label Test' });
+    const label = wrapper.find('.unnnic-checkbox__label__right');
```

Comment:
I would use the `data-testid` attribute to reference the component, since classes have a high possibility of being changed (probably not in this component, but it's good to follow the standard). Here is an example of a [test in Weni Chats using this approach](https://github.com/weni-ai/chats-webapp/blob/main/src/components/chats/__tests__/ModalBulkTransfer.spec.js#L80C1-L80C81).